src/pages/Homepage.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { Routes, Route } from "react-router-dom";
+import { Link } from "react-router-dom";
+import Beers from "./Beers";
+import NewBeer from "./NewBeer";
+
+function Homepage() {
+  return (
+    <div>
+      <div>
+        <img src="./assets/beers.png" alt="Beers" />
```

Comment:
In React to use an image you need to import it first as a variable and then use it in the img tag src:
```
( ... )

import homeImg from "../assets/beers.png"

function Homepage() {
  return (
    <div>
      <div>
        <img src={homeImg} />

( ... )
```